File: 第二次作業/Date.cpp
#include "Date.h"

Date::Date(int m, int d, int y)
{
	month = m;
	day = d;
	year = y;
}

void Date::setMonth(int newM)
{
	month = newM;
}

void Date::setDay(int newD)
{
	day = newD;
}

void Date::setYear(int newY)
{
	year = newY;
}

int Date::getMonth() const
{
	return month;
}

int Date::getDay() const
{
	return day;
}

int Date::getYear() const
{
	return year;
}

File: python練習/StudentCard_Composition/Date.py
class Date:
    def __init__(self, m, d, y):
        self.__month = m
        self.__day = d
        self.__year = y
        
    def setMonth(self, newMonth):
        self.__month = newMonth
        
    def setDay(self, newDay):
        self.__day = newDay
        
    def setYear(self, newYear):
        self.__year = newYear
        
    def getMonth(self):
        return self.__month
    
    def getDay(self):
        return self.__day
    
    def getYear(self):
        return self.__year
    
    def toString(self):
        print(str(self.getMonth()) + '\\' + str(self.getDay()) + '\\' + str(self.getYear()), end=' ')